src/js/player.js

```diff
@@ -1328,6 +1328,16 @@ class Player extends Component {
     return this.tech_;
   }
 
+  /**
+   * Return a Video.js version. Follows [semantic versioning](https://semver.org/).
```

Comment:
A small suggestion `Returns video.js version`. I'm not sure if it's necessary to add `Follows [semantic versioning](https://semver.org/).`

Let's keep it that way and see what @gkatsev thinks.

Reply:
changed https://github.com/videojs/video.js/pull/8543/commits/bd2023040f5d9c6ae2c15e96c8383b9cbf5c5bac